src/tripCardSample/TripCardSample.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components'
+
+import { space } from '../_utils/branding'
+
+export const StyledTripCardSample = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  flex-wrap: wrap;
+
+  .kirk-tripCardSample-price {
```

Comment:
Since this is a brand new component we should use styled-components syntax to avoid targeting HTML elements and stress incapsulation
ex: `.kirk-tripCardSample-price span`

Reply:
Sorry I merged before I realized there were new comments. I made the changes in a new PR: https://github.com/blablacar/ui-library/pull/1076